cmd/server/command/start.go

```diff
@@ -357,6 +380,44 @@ func NewStart(startOptions *startOptions) *cobra.Command {
 				done <- errors.WithMessage(err, "broker")
 			}()
 
+			if s3storageSvc != nil {
+				sqsHandler := func(msg string) error {
+					var s3event s3storage.S3Event
+					s3event.Unmarshal([]byte(msg))
```

Comment:
The error is not checked here? `Unmarhsal` might fail if the payload for some reason isn't what we expect.
```suggestion
					err := s3event.Unmarshal([]byte(msg))
					if err != nil {
						return errors.WithMessage(err, "unmarshal S3Event")
					}
```

Reply:
oops. Good catch 👍